components/typeahead/typeahead.directive.ts

```diff
@@ -6,13 +6,13 @@ import {
 } from 'angular2/core';
 import {NgModel}from 'angular2/common';
 
+import {Observable, Subject} from 'rxjs/Rx';
```

Comment:
Agreed

On Wed, May 11, 2016, 17:15 Leon Radley <notifications@github.com> wrote:

> In components/typeahead/typeahead.directive.ts
> <https://github.com/valor-software/ng2-bootstrap/pull/99#discussion_r62855597>
> :
>
> > @@ -6,13 +6,13 @@ import {
> >  } from 'angular2/core';
> >  import {NgModel}from 'angular2/common';
> >
> > +import {Observable, Subject} from 'rxjs/Rx';
>
> I don't think we should import the whole rxjs library here.
> It's better to import them separatly, that way we don't include more code
> than we need.
>
> import {Observable} from 'rxjs/Observable';
> import {Subject} from 'rxjs/Subject';
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly or view it on GitHub
> <https://github.com/valor-software/ng2-bootstrap/pull/99/files/1a0646b0d5e9e28ac9335e581083014092d9c7a0#r62855597>
>